mock/src/server.rs

```diff
@@ -33,7 +31,7 @@ impl fmt::Display for MockServeError {
 pub struct MockGraphQLServer {
     logger: Logger,
     query_sink: Option<Sender<Query>>,
-    schema_provider_event_sink: Sender<SchemaProviderEvent>,
+    schema_provider_event_sink: Sender<SchemaEvent>,
```

Comment:
I'd rename this field. Since the SchemaProvider is gone, this could be just `schema_event_sink`.